database/youtube/web-development.json

```diff
@@ -78,5 +86,4 @@
         "subcategory": "web-development",
         "language": "english"
     }
```

Comment:
Don’t forget to add a description! 😊

Reply:
codewithharry already exist in project so I have removed from my PR and other changes you told I have done that